bin/cqlsh.py

```diff
@@ -170,6 +170,12 @@ def find_zip(libprefix):
 from cqlshlib.tracing import print_trace, print_trace_session
 from cqlshlib.util import get_file_encoding_bomsize, trim_if_present
 
+from cqlshlib.geotypes import patch_import_conversion  # nopep8
```

Comment:
I would import this as `patch_geotypes_import_conversion` or similar.

Reply:
done